packages/cli/src/telemetry/resource.js

```diff
@@ -104,6 +109,7 @@ export async function getResources() {
     'env.node_env': process.env.NODE_ENV || null,
     'ci.redwood': !!process.env.REDWOOD_CI,
     'ci.isci': ci.isCI,
+    'platform.gitpod': isGitpod,
```

Comment:
platform - name this. Dev environment is likely better. Change to string of name of place rather than list of booleans.